core-server/src/main/java/org/glassfish/jersey/server/internal/inject/ParamConverterFactory.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2012, 2018 Oracle and/or its affiliates. All rights reserved.
+ * (Portions) Copyright (c) 2018 Payara Foundation and/or its affiliates.
```

Comment:
No (Portions) please

Reply:
What is the legal implication of such a "Portions" line?

Reply:
The "(Portions)" word just makes the Glassfish Copyright plugin check fail. Multiple Copyright lines passes. We have discussed this with the previous Payara PR.